packages/frontend/src/components/AddButton/AddButton.tsx

```diff
@@ -19,10 +19,18 @@ const AddButton: React.FC = () => {
     socket.on("authorized/list", (userList: string[]) => {
       setUsers(userList);
     });
+
+    return () => {
+      socket.off("authorized/list");
```

Comment:
Should specify which exact function we want to remove to avoid side-effects in other parts of the program. Since the function is defined in `socket.on` directly we don't have access to it! We can either refactor, but I think we can do a cheaky lil trick:

```typescript
let fn: Function;
socket.on("authorized/list", fn = (userList: string[]) => { ... });

return () => {
  socket.off("authorized/list", fn);
}
```